### PR TITLE
SHARD-1204: Add Secure Account Storage Support

### PR DESCRIPTION
**Pull request by S0naliThakur**
Linear: https://linear.app/shm/issue/SHARD-1204
Summary: Add Secure Account Storage Support

### CODE REVIEW

File: src/types/account.ts [hunk 1]
```diff
@@ -74,6 +75,7 @@ export enum AccountSearchType {
   ContractStorage,
   ContractCode,
   InternalTxReceipt,
+  // SecureAccount,
```
Comment (aniketdivekar) [inline src/types/account.ts]: why is this added as comment?

Comment (S0naliThakur) [reply src/types/account.ts]: The commented-out SecureAccount type was added to the AccountSearchType enum in account.ts as a placeholder for future secure account search functionality. Removed it in the latest commit

**Review by aniketdivekar — APPROVED** [review]
